src/setup_payload/QRCodeSetupPayloadParser.h

```diff
@@ -28,8 +28,12 @@
 #include <lib/core/CHIPError.h>
 #include <lib/core/TLV.h>
 
+#include <algorithm>
+#include <sstream>
```

Comment:
I don't think we need sstream here. Should be left to C++

Reply:
Er, right you are.  Fixing.